wallet/src/main/java/com/android/identity_credential/wallet/presentation/PresentmentFlow.kt

```diff
@@ -55,6 +55,8 @@ const val TAG = "PresentmentFlow"
  * @param mdocCredential the object containing the [Document] and docType amongst other properties.
  * @param trustPoint if provided, identifies the Verifying party.
  * @param encodedSessionTranscript the bytes of `SessionTranscript` CBOR.
+ * @param showConsentPrompt is [true] by default, else if provided [false] then the Consent Prompt
+ *      will not be shown.
```

Comment:
I would just remove `showConsentPrompt` for now since as per the other comment we don't need to pass false from Credman.